pom.xml

```diff
@@ -149,6 +154,12 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plain-credentials</artifactId>
       <optional>true</optional>
+    </dependency>
+      <!-- Basic Username PrivateKey -->
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>ssh-credentials</artifactId>
+      <version>1.13</version>
```

Comment:
`version` will be used from the `dependencyManagement` section, so isn't needed here

Reply:
so, should I remove this dependency completely ?

Reply:
I do not care about `credentials-binding`, this plugin does not (should not at least) need nor should not depend on it. ... 
This is a *credential provider* and as such can be used for things that are not jobs.  I do not want this to become yet another kitchen sink plugin puling in all manner of dependencies that are not needed for the core use case.